### PR TITLE
[WIP] Issue #4 Check custom module files

### PR DESCRIPTION
**Pull request by lrowang**
Hi @lhridley,

Here's a #4 PR for your review!

I created a directory `acs_master` with the 3 files (.module, .info, and .install) using these Drupal.org resources:
- [Getting started](https://www.drupal.org/node/1074362)
- [Telling Drupal about your module](https://www.drupal.org/node/1075072)
- [Writing .install files](https://www.drupal.org/node/876250)

Re Commenting:
- I put some standard commenting in the acs_master.module file, but PhpStorm also added some default comments -- do these get combined somehow?
- I didn't see any standard comments for the .info and .install files and I wasn't sure what to put in them at this juncture since they're empty!

Talk soon,
Ro

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -94,6 +94,7 @@ Vagrant.configure(2) do |config|
      drush updb
      drush fra -y
      drush cc all
+     drush pm-enable acs_master
```
Comment (lhridley) [inline Vagrantfile]: Add the -y parameter after this command

File: www/sites/all/modules/custom/acs_master/acs_master.install [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?php
+/**
```
Comment (lhridley) [inline www/sites/all/modules/custom/acs_master/acs_master.install]: The PHP Storm auto comment needs to be removed, and a file comment added that meets Drupal Coding Standards:

https://www.drupal.org/node/1354#file

File: www/sites/all/modules/custom/acs_master/acs_master.module [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?php
+/**
```
Comment (lhridley) [inline www/sites/all/modules/custom/acs_master/acs_master.module]: Remove the PHP Storm autogenerated comment.

File: Vagrantfile (hunk 1 above)
Comment (lrowang) [inline Vagrantfile]: Ah! Thank you!

File: www/sites/all/modules/custom/acs_master/acs_master.install (hunk 2 above)
Comment (lrowang) [inline www/sites/all/modules/custom/acs_master/acs_master.install]: Got it!

File: www/sites/all/modules/custom/acs_master/acs_master.module (hunk 3 above)
Comment (lrowang) [inline www/sites/all/modules/custom/acs_master/acs_master.module]: Ten four!

**Pull request merged**